### DISCUSSION TITLE
average speed in km/h instead of min/km

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#15417] [Q&A] [answered]
How can I change the average-speed in panel from min/km to km/h? (walking-profile)

**Comment by vshcherb** [comment, marked as answer]
Just change speed units in General Settings https://docs.osmand.net/docs/user/widgets/info-widgets#speed

**Reply by ThomasMeister123** [reply]
Thanx. Thats what I looking for. Maybe an idea: implement that you can change the unit within the widget in the panel. So you can have two average speed-widgets with different units in panel (km/h AND min/km)
